### PR TITLE
added localization support

### PR DESCRIPTION
**Pull request by fork-boy**
## Description
Added localization support by using the next-18next package. Created localization for default English language and french. Created there localize JSON inside public/locale folder. On clicking the change local button in UI the default language English text changes to the french text

## Screenshot
- For the English language
Screenshot at 2021-04-12 13-46-26

- For french language
Screenshot at 2021-04-12 13-46-45

## Tests
- Manual test case done 

### Automated test cases added
- NA

### Manual test cases run
The manual test case done, added "hello world" as a title in public/locale/en/common.json localize file and "Bonjour le monde" title in public/locale/fr/common.json localize file.

### Browsers tested
- [ ] Chrome
- [ ] Safari
- [ ] Internet Explorer
- [ ] Firefox

## Platform dependencies
Does this depend on or impact
- [ ] iOS
- [ ] Android
- [ ] API

### CODE REVIEW

**Review by vindru — APPROVED** [review]

**Comment by jjalan** [comment]
@agentraghav - The route should not change to `localhost/fr`.

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
-import '../styles/globals.css'
+import '../styles/globals.css';
+import { appWithTranslation } from 'next-i18next';
+import type { AppProps } from 'next/app';
 
-function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
-}
+const MyApp = ({ Component, pageProps }: AppProps) => (
```
Comment (jjalan) [inline pages/_app.tsx]: Use React functional component

**Review by jjalan — CHANGES_REQUESTED** [review]
url should not change because the language changed.

File: pages/_app.tsx (hunk 1 above)
Comment (fork-boy) [reply pages/_app.tsx]: done

**Pull request merged**